agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/telemetry/TelemetryClient.java

```diff
@@ -329,9 +327,8 @@ public String getInstrumentationKey() {
   }
 
   @Nullable
-  public String getStatsbeatInstrumentationKey() {
-    StatsbeatConnectionString val = this.statsbeatConnectionString;
-    return val != null ? val.getInstrumentationKey() : null;
+  public StatsbeatConnectionString getStatsbeatConnectionString() {
```

Comment:
we might want to change the statsbeat config from iKey, endpoint to connectionstring only? 
we currently use both ikey and endpoint. It's not a customer facing config.
can you add a todo for me there?